### PR TITLE
V1.1.1 compliance

### PR DESCRIPTION
**Pull request by a-frantz**
title. Branched off of #124 so please let's get that merged first.
1) bc of Rust `wdl` (we need a name) we can be confident I did this properly despite it being manual!
2) Really looking forward to having an autoformatter to do these tasks in the future, because this was **tedious**.

### CODE REVIEW

File: tools/samtools.wdl [hunk 1]
```diff
@@ -719,11 +719,11 @@ task collate_to_fastq {
...
         description: "Runs `samtools collate` on the input BAM file then converts it into FASTQ(s) using `samtools fastq`"
         outputs: {
-            collated_bam: "A collated BAM (reads sharing a name next to each other, no other guarantee of sort order)"
-            read_one_fastq_gz: "Gzipped FASTQ file with 1st reads in pair"
-	        read_two_fastq_gz: "Gzipped FASTQ file with 2nd reads in pair"
-            singleton_reads_fastq_gz: "Gzipped FASTQ containing singleton reads"
-            interleaved_reads_fastq_gz: "Interleaved gzipped paired-end FASTQ"
+            collated_bam: "A collated BAM (reads sharing a name next to each other, no other guarantee of sort order)",
+            read_one_fastq_gz: "Gzipped FASTQ file with 1st reads in pair",
+	        read_two_fastq_gz: "Gzipped FASTQ file with 2nd reads in pair",
```
Comment (adthrasher) [inline tools/samtools.wdl:724]: Is this spacing actually off or is it just the GitHub rendering?

Comment (adthrasher) [inline tools/samtools.wdl:724]:
```suggestion
            read_one_fastq_gz: "Gzipped FASTQ file with 1st reads in pair",
            read_two_fastq_gz: "Gzipped FASTQ file with 2nd reads in pair",
```

File: workflows/qc/markdups-post.wdl [hunk 2]
```diff
@@ -18,13 +18,13 @@ workflow markdups_post {
...
-            mosdepth_global_dist: "Distribution of whole genome coverage produced by `mosdepth`"
-            mosdepth_region_summary: "Summaries of coverage corresponding to the regions defined by `coverage_beds` input, produced by `mosdepth`"
-            mosdepth_region_dist: "Distributions of coverage corresponding to the regions defined by `coverage_beds` input, produced by `mosdepth`"
+            insert_size_metrics: "`*.txt` output file of `picard collectInsertSizeMetrics`",
+            insert_size_metrics_pdf: "`*.pdf` output file of `picard collectInsertSizeMetrics`",
+            flagstat_report: "`samtools flagstat` report",
+            mosdepth_global_summary: "Summary of whole genome coverage produced by `mosdepth`",
+            mosdepth_global_dist: "Distribution of whole genome coverage produced by `mosdepth`",
+            mosdepth_region_summary: "Summaries of coverage corresponding to the regions defined by `coverage_beds` input, produced by `mosdepth`",
+            mosdepth_region_dist: "Distributions of coverage corresponding to the regions defined by `coverage_beds` input, produced by `mosdepth`",
```
Comment (adthrasher) [inline workflows/qc/markdups-post.wdl:27]: For consistency, we haven't included a trailing comma on the final entry.
```suggestion
            mosdepth_region_dist: "Distributions of coverage corresponding to the regions defined by `coverage_beds` input, produced by `mosdepth`"
```

File: workflows/qc/quality-check-standard.wdl [hunk 3]
```diff
@@ -20,53 +20,53 @@ workflow quality_check {
...
+            mosdepth_dups_marked_region_summary: "A summary of mean depths per chromosome and within specified regions per chromosome. There will be one file in this array for each `coverage_beds` input file. This file is produced from analyzing the duplicate marked BAM (only present if `mark_duplicates = true`).",
+            inferred_strandedness: "TSV file containing the `ngsderive strandedness` report",
+            qualimap_rnaseq_results: "Gzipped tar archive of all QualiMap output files",
+            junction_summary: "TSV file containing the `ngsderive junction-annotation` summary",
+            junctions: "TSV file containing a detailed list of annotated junctions",
+            IntermediateFiles: "Any and all files produced as intermediate during pipeline processing. Only output if `output_intermediate_files = true`.",
```
Comment (adthrasher) [inline workflows/qc/quality-check-standard.wdl:69]:
```suggestion
            IntermediateFiles: "Any and all files produced as intermediate during pipeline processing. Only output if `output_intermediate_files = true`."
```

File: tools/cellranger.wdl [hunk 4]
```diff
@@ -91,7 +91,7 @@ task count {
         cpu: ncpu
         memory: "~{memory_gb} GB"
         disk: "~{disk_size_gb} GB"
-        container: "ghcr.io/stjudecloud/cellranger:1.1.1"
+        container: "docker://ghcr.io/stjudecloud/cellranger:1.1.1"
```
Comment (adthrasher) [inline tools/cellranger.wdl:94]:
```suggestion
        container: 'docker://ghcr.io/stjudecloud/cellranger:1.1.1'
```

For consistency, though the `"` was likely mine.

File: tools/cellranger.wdl [hunk 5]
```diff
@@ -153,7 +153,7 @@ task bamtofastq {
...
```
Comment (adthrasher) [inline tools/cellranger.wdl:156]:
```suggestion
        container: 'docker://ghcr.io/stjudecloud/cellranger:1.1.1'
```

**Review by adthrasher — CHANGES_REQUESTED** [review]

File: tools/samtools.wdl (hunk 1 above)
Comment (a-frantz) [reply tools/samtools.wdl:724]: Good catch. I am guessing that CoPilot inserted that tab, which it does sometimes. Very frustrating.

File: tools/cellranger.wdl (hunk 4 above)
Comment (a-frantz) [reply tools/cellranger.wdl:94]: Agree they should all match. But IDK why we use all `"` except for the `container` string. I'd like to make us use double quotes consistently, but that's a more annoying task to perform. I'll probably make it a rule for the future auto-formatter and use that on the repo.

Until then, this should be single quotes to match everywhere else.

File: workflows/qc/markdups-post.wdl (hunk 2 above)
Comment (a-frantz) [reply workflows/qc/markdups-post.wdl:27]: Another good catch.

**Review by adthrasher — APPROVED** [review]

**Pull request merged**